### PR TITLE
Fix CI node_modules caching issue #847

### PR DESCRIPTION
**Pull request by osm-alt**
Added the node_modules path to the caching action of the workflow for fe1-web.

### CODE REVIEW

**Comment by osm-alt** [comment]
Removed caching of node_modules on MacOS due to my previous comment but left it for the other platforms

**Review by 4xiom5 — DISMISSED** [review]
Seems good

File: .github/workflows/ci.yaml [hunk 1]
```diff
@@ -53,6 +53,16 @@ jobs:
             ${{ runner.os }}-build-
             ${{ runner.os }}-
 
+      - name: Setup node modules cache
+        if: ${{ runner.os != 'macOS'}}
```
Comment (Tyratox) [inline .github/workflows/ci.yaml:57]: Why exclude MacOs? Maybe add a comment?

**Review by Tyratox — CHANGES_REQUESTED** [review]
Maybe add a small comment/explanation why macOS is excluded, otherwise LGTM!

**Review by Tyratox — APPROVED** [review]

**Pull request merged**